File: RPC/RPC_10_12_10_2024/Soluciones/ThirteenthAnnual_UCF_LocalProgrammingContest/local2024FinalRoundData/solutions/A_adapter/adapter.cpp
// Arup Guha
// 8/19/2024
// Alternate Solution for 2024 UCF Locals Problem: Christmas Tree Adapter

using namespace std;
#include <iostream>

int main() {

    // Read in info.
    int amp, watt, volt;
    cin >> amp >> watt >> volt;

    // This is what we have to check.
    if (amp*volt <= watt)
        cout << "1" << endl;
    else
        cout << "0" << endl;

    return 0;
}

File: RPC/RPC_10_12_10_2024/Soluciones/ThirteenthAnnual_UCF_LocalProgrammingContest/local2024FinalRoundData/solutions/A_adapter/adapter.py
# Arup Guha
# 8/19/2024
# Alternate Solution to 2024 UCF Locals Problem: Christmas Tree Adapter

# Get input.
amp = int(input())
toks = input().split()
watt = int(toks[0])
volt = int(toks[1])

# What we're checking for.
if watt//volt >= amp:
    print(1)
else:
    print(0)
